site/content/docs/main/tilt.md

```diff
@@ -0,0 +1,180 @@
+`tilt-provider.json` file describing how to build the provider. 
+
+**enable_providers** (Array[]String, default=[]): A list of the provider plugins to enable. See [provider plugins](#provider-plugins)
+for more details. Note: when not making changes to a plugin, it is not necessary to load them into
+Tilt: an existing image and version might be specified in the Velero deployment instead, and Tilt will load that. 
+
+**allowed_contexts** (Array, default=[]): A list of kubeconfig contexts Tilt is allowed to use. See the Tilt documentation on
+*[allow_k8s_contexts](https://docs.tilt.dev/api.html#api.allow_k8s_contexts) for more details. Note: Kind is automatically allowed.
+
+**enable_restic** (Bool, default=false): Indicate whether to deploy the restic Daemonset. If set to `true`, Tilt will look for a `velero/tilt-resources/restic.yaml`  file
+containing the configuration of the Velero restic DaemonSet.
+
+**create_backup_locations** (Bool, default=false): Indicate whether to create one or more backup storage locations. If set to `true`, Tilt will look for a `velero/tilt-resources/velero_v1_backupstoragelocation.yaml` file
+containing at least one configuration for a Velero backup storage location.
+
+**setup-minio** (Bool, default=false): Configure this to  `true` if you want to configure backup storage locations in a Minio instance running inside your cluster. 
+
+### Create Kubernetes resource files to deploy
+All needed Kubernetes resource files are provided as ready to use samples in the `velero/tilt-resources/examples` directory. You only have to move them to the `velero/tilt-resources` level. 
+
+Because the Velero Kubernetes deployment as well as the restic DaemonSet contain the configuration
+for any plugin to be used, files for these resources are expected to be provided by the user so you may choose
+which provider plugin to load as a init container. Currently, the sample files provided are configured with all the
+plugins supported by Velero, feel free to remove any of them as needed.
+
+For Velero to operate fully, it also needs at least one backup
+storage location. A sample file is provided that needs to be modified with the specific
+configuration for your object storage. See the next sub-section for more details on this.
+
+### Configure a backup storage location
+You will have to configure the `backupstoragelocation.yaml` with the proper values according to your storage provider. Read the [plugin documentation](https://velero.io/plugins/)
+to learn what field/value pairs are required for your particular provider's backup storage location configuration.
+
+Below are some ways to configure a backup storage location for Velero.
+#### As a storage with a service provider
+Follow the provider documentation to provision the storage. We have a [list of all known object storage providers](https://velero.io/docs/main/supported-providers/) with corresponding plugins for Velero.
```

Comment:
These should be relative links rather than links including the branch name. I noticed that the docs for 1.5 include `main` in the link. There are a few more links that should be updated on lines 130, 136, 158.